mypy/checker.py

```diff
@@ -1012,7 +1015,29 @@ def is_trivial_body(self, block: Block) -> bool:
             return True
         elif len(body) > 1:
             return False
+
         stmt = body[0]
+
+        # Bodies that just raise "NotImplementedError()" count as trivial.
```

Comment:
Add a doc string for this function, maybe by moving some of the inline comments to it?